rq/queue.py

```diff
@@ -111,6 +111,12 @@ def empty(self):
         script = self.connection.register_script(script)
         return script(keys=[self.key])
 
+    def delete(self):
```

Comment:
Can we add a boolean `delete_jobs` parameter? `queue.empty()` should only be called with `delete_jobs=True`.

Reply:
OK do you want it to have a default parameter value of True or False?